Nvidia_JTX1/Ubuntu/10_setup_ros.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# this script should be run as the apsync user
+# these instructions come from Dan Pollock, posting can be found here: https://www.facebook.com/groups/pixhawk2/permalink/1192680427481244/
+
+set -e
+set -x
+
+pushd .
+
+# setup reference to ros packages
+sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-key 421C365BD9FF1F717815A3895523BAEEB01FA116
+sudo apt-get update
+
+# install ros
+sudo apt-get install -y ros-kinetic-ros-base python-rosinstall ros-kinetic-mavros ros-kinetic-rtabmap-ros ros-kinetic-robot-state-publisher
+sudo c_rehash /etc/ssl/certs
+sudo rosdep init
+rosdep update
+
+# add environment to .bashrc
+echo "source /opt/ros/kinetic/setup.bash" >> ~/.bashrc
+source ~/.bashrc
+
+# install the ZED SDK using 7_setup_zed.sh
+
+# install zed-ros-wrapper
+mkdir -p ~/catkin_zed/src
+cd ~/catkin_zed/src
+git clone https://github.com/stereolabs/zed-ros-wrapper.git
+
+# create workspace
+# Note: this seems to fail because some ROS variables are not being set
+cd ~/catkin_zed/src
+/opt/ros/kinetic/bin/catkin_init_workspace
```

Comment:
this isn't really need normally, the first call of catkin_make will init the workspace